### ISSUE TITLE
`get' on null Nullable!bool is still here

### ISSUE THREAD

**Issue by denizzzka** [nexor/telega#53]
```
core.exception.AssertError@/usr/include/dmd/phobos/std/typecons.d(2971): Called `get' on null Nullable!bool.
----------------
??:? _d_assert_msg [0x5631f348af22]
/usr/include/dmd/phobos/std/typecons.d:2971 inout pure nothrow ref @property @nogc @safe inout(bool) std.typecons.Nullable!(bool).Nullable.get() [0x5631f3326e35]
../../.dub/packages/asdf-0.4.6/asdf/source/asdf/serialization.d:2521 void asdf.serialization.deserializeValue!(telega.botapi.Chat).deserializeValue(asdf.asdf.Asdf, ref telega.botapi.Chat) [0x5631f333019f]
../../.dub/packages/asdf-0.4.6/asdf/source/asdf/serialization.d:2521 void asdf.serialization.deserializeValue!(telega.botapi.Message).deserializeValue(asdf.asdf.Asdf, ref telega.botapi.Message) [0x5631f332fa39]
../../.dub/packages/asdf-0.4.6/asdf/source/asdf/serialization.d:2521 void asdf.serialization.deserializeValue!(telega.botapi.BotApi.MethodResult!(telega.botapi.Message).MethodResult).deserializeValue(asdf.asdf.Asdf, ref telega.botapi.BotApi.MethodResult!(telega.botapi.Message).MethodResult) [0x5631f333b034]
../../.dub/packages/asdf-0.4.6/asdf/source/asdf/serialization.d:434 telega.botapi.BotApi.MethodResult!(telega.botapi.Message).MethodResult asdf.serialization.deserialize!(telega.botapi.BotApi.MethodResult!(telega.botapi.Message).MethodResult).deserialize(asdf.asdf.Asdf) [0x5631f333af02]
../../.dub/packages/asdf-0.4.6/asdf/source/asdf/serialization.d:481 telega.botapi.BotApi.MethodResult!(telega.botapi.Message).MethodResult asdf.serialization.deserialize!(telega.botapi.BotApi.MethodResult!(telega.botapi.Message).MethodResult).deserialize(const(char[])) [0x5631f333aec3]
../../.dub/packages/telega-0.0.8/telega/source/telega/botapi.d:1757 telega.botapi.Message telega.botapi.BotApi.callMethod!(telega.botapi.Message, telega.botapi.SendMessageMethod).callMethod(telega.botapi.SendMessageMethod) [0x5631f333ad33]
../../.dub/packages/telega-0.0.8/telega/source/telega/botapi.d:1828 telega.botapi.Message telega.botapi.BotApi.sendMessage(ref telega.botapi.SendMessageMethod) [0x5631f33253bb]
../../.dub/packages/telega-0.0.8/telega/source/telega/botapi.d:1823 telega.botapi.Message telega.botapi.BotApi.sendMessage!(long).sendMessage(long, immutable(char)[]) [0x5631f32991ca]
source/app.d:81 void app.sendNotifies(paper_plane_bot.grab.PackageDescr[]) [0x5631f328070b]
source/app.d:44 _Dmain [0x5631f328035b]
```

**Comment by nexor** [comment]
Please provide incoming JSON for Message type

**Comment by nexor** [comment]
Reproduced https://github.com/libmir/asdf/issues/126

**Comment by nexor** [comment]
Closed in #56
